CODEOWNERS

```diff
@@ -557,6 +557,7 @@
 /samples/wifi/promiscuous/                @D-Triveni
 /samples/wifi/wfa_qt_app/                 @D-Triveni
 /samples/wifi/offloaded_raw_tx/           @kapbh @sachinthegreen
+/samples/untitled_sample_demo/            @nrfconnect/ncs-co-build-system
```

Comment:
Another entry for the sample docs below probably in line 674:
 /samples/untitled_sample_demo/*.rst            @nrfconnect/ncs-vestavind-doc. Also, since it is a new sample, a changelog entry would be nice.